code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -340,3 +343,6 @@
 
 /obj/machinery/atmospherics/proc/update_layer()
 	layer = initial(layer) + (piping_layer - PIPING_LAYER_DEFAULT) * PIPING_LAYER_LCHANGE
+
+/obj/machinery/atmospherics/proc/paint(paint_color)
+	return FALSE
```

Comment:
This is to prevent mixers and such from being painted right?

Reply:
If so, why have both this and the FALSE paintable by default?

Reply:
That's because the `paint` implementation requires some methods that are only available to `pipe`, so we cannot put this logic here.